### DISCUSSION TITLE
mkl repository

### DISCUSSION THREAD

**Discussion by rootsmusic** [oneapi-src/oneMKL#340] [Q&A] [answered]
Is this the right repo to open issues about [mkl](https://pypi.org/project/mkl/)?  Thanks.

**Comment by mmeterel** [comment, marked as answer]
@rootsmusic  The link you provided indicates Intel oneMKL product that is shipped in binary. 
Please use this [link](https://community.intel.com/t5/Intel-oneAPI-Math-Kernel-Library/bd-p/oneapi-math-kernel-library) for questions on it.
